packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx

```diff
@@ -1,17 +1,48 @@
+import { useEffect } from 'react'
+
 import {
   accountSelectors,
   decodeHashId,
   formatMessageDate
 } from '@audius/common'
 import type { ChatMessage } from '@audius/sdk'
 import { View } from 'react-native'
+import Svg, { Path, Rect } from 'react-native-svg'
 import { useSelector } from 'react-redux'
 
+// import { ReactComponent as ChatTail } from 'app/assets/images/ChatTail.svg'
 import { Text } from 'app/components/core'
 import { makeStyles } from 'app/styles'
+import { useThemePalette } from 'app/utils/theme'
 
 const { getUserId } = accountSelectors
 
+const messages = {
+  newMessage: 'New Message',
+  s: 's'
+}
+
+const ChatTail = ({ fill }) => (
```

Comment:
Having trouble when importing from svg file, this is the only way I've got it to work so far. Any ideas on what's going wrong here?

Reply:
Was importing incorrectly!